docs/sources/integrations/grafana-alerting/index.md

```diff
@@ -21,38 +21,31 @@ weight: 100
 
 Grafana Alerting for Grafana OnCall can be set up using two methods:
 
-- Grafana Alerting: Grafana OnCall is connected to the same Grafana instance being used to manage Grafana OnCall.
-- Grafana (Other Grafana): Grafana OnCall is connected to one or more Grafana instances, separate from the one being used to manage Grafana OnCall.
+- Grafana OnCall is connected to the same Grafana instance being used to manage Grafana OnCall.
+- Grafana OnCall is connected to one or more Grafana instances, separate from the one being used to manage Grafana OnCall.
 
-## Configure Grafana Alerting for Grafana OnCall
-
-You must have an Admin role to create integrations in Grafana OnCall.
-
-1. In the **Integrations** tab, click **+ New integration to receive alerts**.
-2. Select **Grafana Alerting** by clicking the **Quick connect** button or select **Grafana (Other Grafana)** from
-   the integrations list.
-3. Follow the configuration steps that display in the **How to connect** window to retrieve your unique integration URL
-   and complete any necessary configurations.
-
-### Configure Grafana Cloud Alerting
+## Configure Grafana Alerting in same Grafana instance
 
 Use the following method if you are connecting Grafana OnCall with alerts coming from the same Grafana instance from
 which Grafana OnCall is being managed.
 
 1. In Grafana OnCall, navigate to the **Integrations** tab and select **New Integration to receive alerts**.
-1. Click **Quick connect** in the **Grafana Alerting** tile. This will automatically create the integration in Grafana
-   OnCall as well as the required contact point in Alerting.
+2. Click **Quick connect** in the **Grafana Alerting** tile. This will open a **New Grafana Alerting integration** configuration window.
```

Comment:
```suggestion
1. Click **Quick connect** in the **Grafana Alerting** tile. This will open a **New Grafana Alerting integration** configuration window.
```

Keep every item number as 1. It will be substituted by CMS